apps/E2E/src/Menu/pages/MenuPageObject.ts

```diff
@@ -70,57 +70,25 @@ class MenuPageObject extends BasePage {
     );
   }
 
+  /* When menu item 1 is clicked, it increments a counter variable in the test component page, which is displayed in a text box. */
```

Comment:
Add reasoning - Why do we need this? What value is added by incrementing a counter variable?